pkg/translate/translate.go

```diff
@@ -445,6 +481,15 @@ func (t *Translator) setEnablementAndNamespaces(root map[string]interface{}, icp
 	return nil
 }
 
+// IsFeatureEnabled reports whether the feature with name ft is enabled, according to the translations in t,
```

Comment:
I'm not sure if this is necessary - IsComponentEnabled takes feature enablement into account and we don't really check enablement at feature level on its own. Unless you have a specific use for this in mind I recommend leaving it out.

Reply:
I tried to figure out a solution to reuse the `IsComponentEnabled` function during initing the  `IstioControlPlane`, https://github.com/istio/operator/blob/9027646f6009e6b39135ca7505cf1d9f27ef8aea/pkg/component/controlplane/control_plane.go#L39-L45
Then I found that the most efficient method is to add `IsFeatureEnabled` function.